autocapture.md

```diff
@@ -0,0 +1,36 @@
+This is a rough proposal that fits into the category of complex product decisions. I will polish this markdown as a PR.
+
+# Proposal: stop supporting autocapture
+
+We currently support autocapture of front end events.
+
+This helps us with:
+
+* Users can get events in more quickly
+* It captures lots of useful pageview data
```

Comment:
Don't underestimate how valuable this is for e.g. documentation sites like kea. With no setup I can see what links are being clicked and what aren't.